### DISCUSSION TITLE
Should I just use Vite only when using Wagmi?

### DISCUSSION THREAD

**Discussion by hskang9** [WalletConnect/walletconnect-monorepo#4154] [Developer Support] [answered]
Why does this project claim it works with wagmi when it only supports Vite?
While React now requires to be used with Next.js, why are they only supporting Vite and fading it away?

https://github.com/wevm/wagmi/issues/3488

**Comment by jxom** [comment, marked as answer]
Wagmi does support Next.js, and any other React framework…

Feel free to get started by running `npm init wagmi` and choosing “Next.js” ☺️
